tools/validate.js

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env node
+/*eslint-env node*/
+"use strict";
+const { exec } = require("child_process");
+const handler = require("serve-handler");
+const http = require("http");
+
+/**
+ *
+ * @param {string} cmd A string representing a shell command.
+ */
+function toExecutable(cmd) {
```

Comment:
This name is a bit confusing. ("It's generating an executable file?")

How about CommandRunner? Thus, `CommandRunner(cmd).run()`.

Reply:
Good suggestion. Will change.

Reply:
Will just make it a class called `ShellCommand`.